src/middlewared/middlewared/utils/db.py

```diff
@@ -0,0 +1,5 @@
+def dict_factory(cursor, row):
```

Comment:
We can have a common function for connecting to SQLite and querying config row as well